build.gradle.kts

```diff
@@ -48,45 +48,27 @@ tasks.register("setupGitHooks") {
         }
 
         when {
-            currentHooksPath == "scripts" -> {
+            currentHooksPath == "scripts/git-hooks" -> {
                 println("✅ Git hooks already configured correctly")
             }
-            currentHooksPath.isEmpty() -> {
-                // Configure Git to use scripts directory for hooks
+            else -> {
+                // Always configure to use scripts/git-hooks directory
                 exec {
-                    commandLine("git", "config", "core.hooksPath", "scripts")
+                    commandLine("git", "config", "core.hooksPath", "scripts/git-hooks")
                 }
-                println("🔧 Configured Git to use scripts/ directory for hooks")
-            }
-            else -> {
-                println("⚠️ Git hooks path is currently set to: $currentHooksPath")
-                println("💡 To use project hooks, run: git config core.hooksPath scripts")
+                println("🔧 Configured Git to use scripts/git-hooks/ directory for hooks")
             }
         }
 
         // Make hook scripts executable
         val hookFiles = listOf("pre-commit", "pre-push", "commit-msg")
```

Comment:
We should iterate all the files in the scripts/git-hooks directory if all we have there are the hooks.